README.md

```diff
@@ -1,2 +1,23 @@
 # Portfolio
 Microverse portfolio project
+
+## Built With
+
+-   HTML, CSS
+-   linters
+
+## Authors
```

Comment:
- [ ] Kindly work more on your README file, you can use [Microverse README template](https://github.com/microverseinc/readme-template). **Getting started** section is missing. 👍